2018-12-06 14:23:39,545 ERROR: Exception on /explore [GET] [in c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py:1760]
Traceback (most recent call last):
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 2291, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1814, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1812, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1798, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Projects\AppServer\FlaskApp\app\routes.py", line 156, in explore
    page, app.config['POSTS_PER_PAGE'], False)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_sqlalchemy\__init__.py", line 485, in paginate
    if per_page < 0:
TypeError: '<' not supported between instances of 'UnboundField' and 'int'
2018-12-06 14:23:40,147 ERROR: Exception on /index [GET] [in c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py:1760]
Traceback (most recent call last):
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 2291, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1814, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1812, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1798, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Projects\AppServer\FlaskApp\app\routes.py", line 36, in index
    page, app.config['POSTS_PER_PAGE'], False)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_sqlalchemy\__init__.py", line 485, in paginate
    if per_page < 0:
TypeError: '<' not supported between instances of 'UnboundField' and 'int'
2018-12-06 14:23:41,254 ERROR: Exception on /index [GET] [in c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py:1760]
Traceback (most recent call last):
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 2291, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1814, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1812, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1798, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Projects\AppServer\FlaskApp\app\routes.py", line 36, in index
    page, app.config['POSTS_PER_PAGE'], False)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_sqlalchemy\__init__.py", line 485, in paginate
    if per_page < 0:
TypeError: '<' not supported between instances of 'UnboundField' and 'int'
2018-12-06 14:25:15,205 ERROR: Exception on /explore [GET] [in c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py:1760]
Traceback (most recent call last):
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 2291, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1814, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1812, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1798, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Projects\AppServer\FlaskApp\app\routes.py", line 156, in explore
    page, app.config['POSTS_PER_PAGE'], False)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_sqlalchemy\__init__.py", line 485, in paginate
    if per_page < 0:
TypeError: '<' not supported between instances of 'UnboundField' and 'int'
2018-12-06 14:25:17,627 ERROR: Exception on /user/chrisk0208 [GET] [in c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py:1760]
Traceback (most recent call last):
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 2291, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1814, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1812, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask\app.py", line 1798, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Projects\AppServer\FlaskApp\app\routes.py", line 92, in user
    page, app.config['POSTS_PER_PAGE'], False)
  File "c:\users\rtcjk\appdata\local\programs\python\python36-32\lib\site-packages\flask_sqlalchemy\__init__.py", line 485, in paginate
    if per_page < 0:
TypeError: '<' not supported between instances of 'UnboundField' and 'int'
